home.admin/config.scripts/bonus.rtl.sh

```diff
@@ -180,7 +180,7 @@ if [ "$1" = "1" ] || [ "$1" = "on" ]; then
     # install
     echo "# Running npm install ..."
     export NG_CLI_ANALYTICS=false
-    sudo -u rtl npm install --only=prod --logLevel warn
+    sudo -u rtl npm install --legacy-peer-deps --only=prod --logLevel warn
```

Comment:
on the latest release the recommendation is:
```
npm install --omit=dev
```
https://github.com/Ride-The-Lightning/RTL/releases/tag/v0.13.0

```suggestion
    sudo -u rtl npm install --omit=dev
```